### PR TITLE
fix: Add missing null checks

### PR DESCRIPTION
**Pull request by provokateurin**

### CODE REVIEW

File: lib/Command/FolderCommand.php [hunk 1]
```diff
@@ -40,7 +40,14 @@ protected function getFolder(InputInterface $input, OutputInterface $output): ?a
 			return null;
 		}
 
-		$folder = $this->folderManager->getFolder($folderId, $this->rootFolder->getMountPoint()->getNumericStorageId());
+		$rootStorageId = $this->rootFolder->getMountPoint()->getNumericStorageId();
+		if ($rootStorageId === null) {
+			$output->writeln('<error>storage id not found ' . $rootStorageId . '</error>');
```
Comment (come-nc) [inline lib/Command/FolderCommand.php:45]: You are concatenating with null here, not really useful.
```suggestion
			$output->writeln('<error>Root storage id not found</error>');
```

File: lib/Command/ListCommand.php [hunk 2]
```diff
@@ -55,6 +55,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 		}
 
 		$rootStorageId = $this->rootFolder->getMountPoint()->getNumericStorageId();
+		if ($rootStorageId === null) {
+			$output->writeln('<error>storage id not found ' . $rootStorageId . '</error>');
```
Comment (come-nc) [inline lib/Command/ListCommand.php:59]: same

File: lib/Controller/FolderController.php [hunk 3]
```diff
@@ -73,7 +78,12 @@ private function formatFolder(array $folder): array {
 	#[NoAdminRequired]
 	#[ApiRoute(verb: 'GET', url: '/folders')]
 	public function getFolders(bool $applicable = false): DataResponse {
-		$folders = $this->manager->getAllFoldersWithSize($this->getRootFolderStorageId());
+		$storageId = $this->getRootFolderStorageId();
+		if ($storageId === null) {
+			throw new OCSNotFoundException();
```
Comment (come-nc) [inline lib/Controller/FolderController.php:83]: Should be documented no? Isn’t the openapi extractor checking that?

Comment (provokateurin) [reply lib/Controller/FolderController.php:83]: Yeah, but I will use a psalm rule for that to enforce documenting all possible exceptions. So expect a follow up for that.

**Review by come-nc — APPROVED** [review]

**Pull request merged**